### ISSUE TITLE
App still maintained?

### ISSUE THREAD

**Issue by IzzySoft** [AnimbusDev/GEM-Player#53]
Almost 3 years have passed since the last commit, so I wonder whether you still maintain this project, @AdrianVovk?

**Comment by AdrianVovk** [comment]
No. The project is unmaintained, and I've moved on to better things. Thank you for your interest

**Comment by IzzySoft** [comment]
Thanks for the fast answer, @AdrianVovk – and good luck with wherever you moved to :smiley: 

But may I suggest you point that out in the README (so it's clearly visible), or even archive the project (can be done (and undone) via the project settings here at Github)?

**Comment by AdrianVovk** [comment]
Yes, good idea

If you're curious, I've moved on to [linux distro development](https://bitbucket.org/carbonOS)
